src/rviz/TSRMarker.cpp

```diff
@@ -0,0 +1,14 @@
+#include <aikido/rviz/TSRMarker.hpp>
+
+using dart::dynamics::SimpleFrame;
+
+namespace aikido {
+namespace rviz {
+
+TSRMarker::TSRMarker(std::vector<std::unique_ptr<SimpleFrame>> tsrFrames)
```

Comment:
> This already doesn't make any copies if the caller uses `std::move`

That is true, but the explicit rvalue argument could prevent passing lvalue by accident (if I'm missing something here). For example:
```cpp
class TSRMarker
{
public:
  TSRMarker(std::vector<std::unique_ptr<SimpleFrame>>&& tsrFrames);
  // ...
};
std::vector<std::unique_ptr<SimpleFrame>> frames = /* ... */;
auto marker1 = TSRMarker{frames}; // error
auto marker2 = TSRMarker{std::move(frames)}; // okay
```